README.md

```diff
@@ -213,3 +214,143 @@ If you'd like to do this process manually, then:
    * `sh RunDoxygen.sh`
    * In newly created `html` folder, open index.html
+
+
+# 7. Adding New Examples<a name="add-examples" />
+
+As described [above](#examples), you can easily run OpenUxAS examples using the `run-example` script.
+You can also easily add new examples to OpenUxAS that will work with the `run-example` script.
+
+There are three essential parts of an example:
+
+1. the directory that contains the example;
+2. the input files that provide example data; and
+3. the YAML file that describes the details of how the example should be configured and run.
+
+## 7.1. Creating an Example Directory
+
+The `run-example` script will run examples from anywhere, provided you specify the path to the directory containing one or more examples by using the `--examples-dir` option.
+By default, `run-example` looks for examples under the `examples` directory in this repository.
+These instructions assume that you will put your new example in the root of `examples`.
+
+The examples provided in the `examples` directory follow a particular naming convention.
+While this convention makes finding and referring to examples simpler, it is not required: you may name your example however you like (although the repository maintainers may request that you follow a particular naming convention).
+
+We'll name our new example `20_NewExample`.
+Create the directory for the example:
+
+    OpenUxAS$ mkdir examples/20_NewExample
+
+## 7.2. Creating the Input Files
+
+The details of the input files required for OpenUxAS and OpenAMASE are beyond the scope of this README.
+The examples included in this repository offer insight and templates that can be copied and specialized for your new example.
+You can also find details in the User Manual, which you can build following the instructions [above](#docs).
+
+For the purposes of this README, we will simply copy the files in `02_Example_WaterwaySearch`.
+Copy the following to your new example:
+
+* `cfg_WaterwaySearch.xml`
+* `Scenario_WaterwaySearch.xml`
+* everything under `MessagesToSend`
+
+That is:
+
+    OpenUxAS/examples$ cp 02_Example_WaterwaySearch/cfg_WaterwaySearch.xml 20_NewExample
+    OpenUxAS/examples$ cp 02_Example_WaterwaySearch/Scenario_WaterwaySearch.xml 20_NewExample
+    OpenUxAS/examples$ cp -R 02_Example_WaterwaySearch/MessagesToSend 20_NewExample
+
+## 7.3. Creating the YAML File
+
+The YAML file, which must be named `config.yaml`, describes the details of how the example should be configured and run.
+Continuing the example from the two sections above, the YAML file would have the following contents:
+
+    amase:
+        scenario: Scenario_WaterwaySearch.xml
+
+    uxas:
+        config: cfg_WaterwaySearch.xml
+        rundir: RUNDIR_WaterwaySearch
+
+Broadly, there are two relevant sections to the YAML file:
+
+1. `amase`: an optional section that specifies that OpenAMASE should be run and that provides configuration parameters for OpenAMASE; and
```

Comment:
If this is optional, it would be useful to state what happens if it is not included.

Reply:
Fixed.